src/irc/ClientPool.ts

```diff
@@ -496,7 +496,7 @@ export class ClientPool {
     public getNickUserIdMappingForChannel(server: IrcServer, channel: string): {[nick: string]: string} {
         const nickUserIdMap: {[nick: string]: string} = {};
         for (const [userId, client] of this.virtualClients[server.domain].userIds.entries()) {
-            if (client.chanList.includes(channel)) {
+            if (client.inChannel(channel)) {
```

Comment:
I'm a bit surprised that this became `.inChannel` instead of `client.chanList.has(channel)`

Reply:
We should have been using inChannel from the start really :/